sru-templates/manual/ec2-sru

```diff
@@ -50,6 +77,8 @@ for series in $SRU_SERIES; do
     EC2_INST=`launch-ec2 --series $series -u sethostname.yaml | awk '/Found/{print $5}'`
     [ -n "$EC2_INST" ] || { echo "Failed to launch ec2 instance!"; exit 1; }
     sleep 60
+
+    trap "terminate_instances ${EC2_INST}" ERR
```

Comment:
EC2_INST has ubuntu@ in it   I think  you might need ${EC2_INST/ubuntu@/}   ?

Reply:
I am performing that substitution on the `get_ec2_instance_id` function. But maybe we should be more explicitly regarding this parameter

Reply:
that's ok, sorry I missed that on initial review. it's good